COLLABORATOR_GUIDE.md

```diff
@@ -125,6 +125,7 @@ Additionally:
 - Except when updating dependencies, all commits should be self
   contained (meaning every commit should pass all tests). This makes
   it much easier when bisecting to find a breaking change.
+- The [core-validate-commit](https://github.com/evanlucas/core-validate-commit#core-validate-commit) command validates the commit message for a particular commit in node core.
```

Comment:
Can you break this line in to two lines and align the second line right below the first one?